en/docs/learn/identifier-first-flow-handler.md

```diff
@@ -0,0 +1,91 @@
+# Identifier-first Flow Handler
+
+The identifier-first login enables identifying the individuals prior to authenticating them. It is used to get the identity of the user without using authentication information, and use that identity to control the authentication flow.
+
+!!! info 
+    The identifier-first flow handler is shipped with the product itself from IS 5.7.0 onwards.
+
+## Configuring Identifier-first handler in the login flow
+
+!!! warning
+    This handler can be configured at any step in the authentication flow. However, it is not an authenticator by itself and needs to be configured along with another authenticator in order for the authentication process to be successful.
+
+1.  Make sure that the validate_username parameter in the IS_HOME/repository/conf/deployment.toml file is set to true if you want your username validated first.
```

Comment:
`validate_username` and `IS_HOME/repository/conf/deployment.toml` should be in monospace